include/proxsuite/helpers/common.hpp

```diff
@@ -27,4 +27,12 @@ struct infinite_bound
 } // helpers
 } // proxsuite
 
+#if __cplusplus >= 201703L
```

Comment:
Should be nice to move to a common file proxsuite/fwd.hpp if possible.

Reply:
Good idea, I created the file and moved the macro definitions there.